### PR TITLE
chore: 🤖 update host catalog ui

### PR DESCRIPTION
**Pull request by DhariniJeeva**
[jira](https://hashicorp.atlassian.net/browse/ICU-5723)

[redirect pr ](https://github.com/hashicorp/boundary/pull/2476/files)

**region updates:**
Screen Shot 2022-09-20 at 12 49 17 PM

[azure](https://boundary-7lvjmssa6-hashicorp.vercel.app/scopes/s_1sheljmjgm/host-catalogs/new?type=azure)

### CODE REVIEW

File: addons/core/translations/form/en-us.yaml [hunk 1]
```diff
@@ -141,15 +141,15 @@ subscription_id:
...
 access_key_id:
   label: Access Key ID
-  help: The access key ID found on the Security Credentials page in AWS.
+  help: The access key ID generated by AWS for the IAM user to use with this host catalog.
 secret_access_key:
   label: Secret Access Key
-  help: The secret access key found on the Security Credentials page in AWS.
+  help: The secret access key generated by AWS for the IAM user to use with this host catalog.
 region:
   label: Region
```
Comment (ZedLi) [inline addons/core/translations/form/en-us.yaml:149]: Is this still being used or can it be removed?

**Review by ZedLi — APPROVED** [review]
LGTM, just left one comment

**Review by cameronperera — APPROVED** [review]
LGTM!
Ditto Zed's comment about the "Region" translation.

File: addons/core/translations/form/en-us.yaml (hunk 1 above)
Comment (DhariniJeeva) [reply addons/core/translations/form/en-us.yaml]: removed! thank you

**Pull request merged**